core/create_spoken_forms.py

```diff
@@ -270,27 +270,31 @@ def create_spoken_forms_from_regex(source: str, pattern: re.Pattern):
 def generate_string_subsequences(
     source: str,
     words_to_exclude: list[str],
-    minimum_term_length=DEFAULT_MINIMUM_TERM_LENGTH,
+    minimum_term_length: int,
 ):
+    # Includes (lower-cased):
+    # 1. Each word in source, eg "foo bar baz" -> "foo", "bar", "baz".
+    # 2. Each leading subsequence of words from source,
+    #    eg "foo bar baz" -> "foo", "foo bar", "foo bar baz"
+    #    (but not "bar baz" - TODO: is this intentional?)
```

Comment:
I believe this was to avoid quadratic explosion back when DFA was more of an issue. Might still be a problem for dragon?